### DISCUSSION TITLE
Old Style Numerals | How to Enable?

### DISCUSSION THREAD

**Discussion by pkazmier** [MonoLisaFont/feedback#283] [Q&A]
Hello! I just purchased the Plus version of MonoLisa tonight. I was going through enabling all of the extra features in my Terminal program (so far I'm liking ss02, ss07, ss11, ss14). I really wanted to try the Old Style numerals. Is that available as an option? If so, what number? Thanks!

**Comment by bebraw** [comment]
The group for Old Style Numerals is called `onum` so you would have to find a way to enable that in your terminal. The name of the group comes from the standard.

I believe another option might be to freeze it to the font through the customize tool at `/orders` since you bought the plus plan.

**Reply by kger** [reply]
I have the same question, about obtaining the "old numbers" option.  I do not see that option on the font customization download page, even though I, too, bought the Plus plan.  Am I just overlooking it?  I see the zero option (between slashed and dotted), but not one for choosing the old numbers style.

**Reply by bebraw** [reply]
I think `onum` might not have been exposed through the customize tool for some reason (maybe a technical limitation or we overlooked something). CC @okonet.

Typically `onum` is used specifically for design and it's visible in programs like Adobe Illustrator. Many editors, like VS Code, also allow enabling it easily and it's worth sending bug reports to editors that don't allow OpenType customization.

**Comment by pkazmier** [comment]
Ah, thank you very much. I didn’t realize that. TIL!

 It might be helpful for others if the Playground had a section for old style numerals with an `onum` label.

￼


It looks and works great in WezTerm. Easy to configure:

  config.font_size = 14.0
  config.line_height = 1.1
  config.harfbuzz_features = {
    "onum",
    "ss02",
    "ss07",
    "ss11",
    "ss14",
  }

Thanks!
Pete
